.claude/skills/tdd/references/patterns.md

```
@@ -0,0 +1,176 @@
+// テスト
+test("1+1は2", () => {
+  expect(add(1, 1)).toBe(2);
+});
+
+// 仮実装
+function add(a: number, b: number): number {
+  return 2; // ハードコード
+}
+```
+
+**いつ使う**:
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)